### DISCUSSION TITLE
Set initial option with vuex

### DISCUSSION THREAD

**Discussion by Jesus82** [sagalbot/vue-select#1508] [Help Wanted]
I have a country selector that works with vuex. Right now I have something like this

    <select-simple
        @input="mutateSelectedValue"
        :options="getSortedCountries"
        :value="getSelectedCountry"
        :label="'name'"
        :selectedcountry="getSelectedCountry"/>

Once I select a country, it works great, but the problem is that when the component first load, if there is an already selected option in vuex, there is no way to make that show. (It will show blank or it will show the placeholder) Which would be the way to make it work?

**Comment by cannap** [comment]
i have the same

**Comment by rizkhal** [comment]
are you have any solution for this?

**Comment by Igfig** [comment]
I think I found an answer to this. If you pass `:modelValue="getSelectedCountry"` instead of `:value="getSelectedCountry"`, it should work correctly.
